src/Plugins/Reminder.php

```diff
@@ -10,28 +10,28 @@ class Reminder extends NuntiusPluginAbstract {
   protected $category = 'Reminders';
 
   public $formats = [
-    '/remind me next time I logging in to (.*)/' => [
+    '/remind me next time I log into (.*)/' => [
```

Comment:
"I logging in" does not sound right to me. I could be wrong. Can you bring Adam into the conversation? He will know.  In any case, there needs to also be a "the" in front of "next time."